src/main/java/kr/codesquad/Main.java

```diff
@@ -2,6 +2,7 @@
 
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+        LadderGame ladderGame = new LadderGame();
+        ladderGame.startGame();
```

Comment:
```suggestion
        ladderGame.start();
```
단어가 겹치지 않게 메소드 이름을 정하는 게 좋습니다.